tfjs-backend-wasm/src/cc/prelu.cc

```diff
@@ -0,0 +1,88 @@
+/* Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ===========================================================================*/
+
+#include <emscripten.h>
+#include <xnnpack.h>
+#include <cmath>
+#include <cstdio>
+#include <map>
+#include <memory>
+#include <vector>
+
+#include "src/cc/backend.h"
+#include "src/cc/util.h"
+
+const int kBlockSize = 48;
+
+namespace {
+// The operator cache maps the weights id to the xnn_operator_t instantiated for
+// // this set of weights.
+std::map<int, xnn_operator_t> operator_cache;
```

Comment:
`std::unordered_map` would be better: `std::map` is typically implemented as a search tree, but `std::unordered_map` is a hash map.